components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/DefaultSessionManager.cs

```diff
@@ -64,6 +63,51 @@ public class DefaultSessionManager : ISessionManager, IDisposable {
             }
         }
 
+        /// <inheritdoc/>
+        public bool IsConnectionOk(ConnectionModel connection) {
+            var key = new ConnectionIdentifier(connection);
+            _lock.Wait();
+            try {
+                if (!_sessions.TryGetValue(key, out var wrapper)) {
+                    return false;
+                }
+                return wrapper.State == SessionState.Running;
+            }
+            finally {
+                _lock.Release();
+            }
+        }
+
+        /// <inheritdoc/>
+        public int GetNumberOfGoodNodes(ConnectionModel connection) {
```

Comment:
@peterbryntesson, this method is not used anywhere. Was it intended to be used in `SubscriptionWrapper` similar to `DefaultSessionManager.IsConnectionOk` ? Can we delete it if it is not referenced anywhere ?

Reply:
Good spotting, Forgot to remove those